gradle/libs.versions.toml

```
@@ -1,14 +1,13 @@
 [versions]
 agp = "8.5.0"
-kotlin = "2.0.0"
-nexus-publish = "2.0.0"
+kotlin = "2.0.20"
 android-minSdk = "24"
 android-compileSdk = "34"
 
 [libraries]
 kotlin-test = { module = "org.jetbrains.kotlin:kotlin-test", version.ref = "kotlin" }
-nexus-publish = { module = "io.github.gradle-nexus.publish-plugin:io.github.gradle-nexus.publish-plugin.gradle.plugin", version.ref = "nexus-publish" }
 
 [plugins]
 androidLibrary = { id = "com.android.library", version.ref = "agp" }
-kotlinMultiplatform = { id = "org.jetbrains.kotlin.multiplatform", version.ref = "kotlin" }
\ No newline at end of file
+kotlinMultiplatform = { id = "org.jetbrains.kotlin.multiplatform", version.ref = "kotlin" }
+publish = { id = "com.vanniktech.maven.publish", version = "0.29.0" }
```

Personally I'd like to have a bit more context about the plugin, and just add a somewhat descriptive 'group' to the name. So if this was my project, I'd rename it to include part of the group:

```suggestion
vanniktech-mavenPublish = { id = "com.vanniktech.maven.publish", version = "0.29.0" }
```

This is a weak suggestion, I'll leave it up to you.